1.9/overview/architecture/index.md

```diff
@@ -0,0 +1,29 @@
+---
+post_title: Architecture
+nav_title: Architecture
+menu_order: 2
+---
+
+DC/OS is a software platform on which user software runs. This platform layer runs on top of the infrastructure layer. The infrastructure layer is composed of virtual or physical hardware including compute, storage, and networking.
```

Comment:
`an infrastructure layer` makes it sound like there's more than one layer, rather than more than one infrastructure provider... It's like the difference between an interface and the service that implements it.